external-crates/move/crates/move-vm-runtime/Cargo.toml

```diff
@@ -22,10 +22,10 @@ smallvec.workspace = true
 
 move-bytecode-verifier = { path = "../move-bytecode-verifier" }
 move-core-types.workspace = true
-move-vm-config.workspace = true
-move-vm-types.workspace = true
+move-vm-config = { workspace = true, features = ["gas-profiler"] }
+move-vm-types = { workspace = true, features = ["gas-profiler"] }
 move-binary-format.workspace = true
-move-vm-profiler.workspace = true
+move-vm-profiler = { workspace = true, features = ["gas-profiler"] }
```

Comment:
I don't believe these need to be set both here and in the feature flags below?

Reply:
see below